Mathlib/Algebra/Polynomial/BigOperators.lean

```diff
@@ -73,6 +73,17 @@ theorem degree_list_sum_le (l : List S[X]) : degree l.sum ≤ (l.map natDegree).
     rw [List.map_eq_nil_iff] at h
     simp [h]
 
+theorem degree_list_sum_le_of_forall_degree_le (l : List S[X])
```

Comment:
If you move this above, can you use it to golf the previous lemma?

Reply:
Also, do we use `list_sum` or `listSum`?

Reply:
I think it's `listSum` now, yeah

Reply:
Yes, seems like it is possible to slightly golf the other lemma if you rearrange them, I've done this now.

Searching mathlib for `listSum` vs `list_sum` yields 7 instances of the former and 76 instances of the latter. If we are going to rename this, perhaps that is better done in the form of a dedicated PR.